services-custom/dynamodb-enhanced/src/main/java/software/amazon/awssdk/enhanced/dynamodb/extensions/VersionedRecordExtension.java

```diff
@@ -144,11 +202,38 @@ public WriteModification beforeWrite(DynamoDbExtensionContext.BeforeWrite contex
 
     @NotThreadSafe
     public static final class Builder {
+        private Long startAt = 0L;
```

Comment:
Is there any reason we are setting the default values in Builder? Builder class should not have any logic ideally and it also seems we are already setting the default values in the top level classes